.buildkite/pipeline.yml

```diff
@@ -7,6 +7,7 @@ env:
 steps:
   - label: 'hydra-eval-errors'
     command: 'nix-build ./nix -A iohkNix.hydraEvalErrors && ./result/bin/hydra-eval-errors.py'
+    if: 'build.tag != null'
```

Comment:
I am not sure to get this one?

Reply:
Oops, I meant the opposite. That script fails unconditionally for tag builds, so I want to disable it. Fixed.